website/snippets/_config-description-resource.md

```diff
@@ -0,0 +1 @@
+Resource-specific configurations are applicable to only one dbt resource type rather than multiple resource types. You can define these settings in the project file (`dbt_project.yml`), a property file (`models/properties.yml` for models, similarly for other resources), or within the resource’s file using the `{{ config() }}` macro.<br /><br /><span>The following resource-specific config code block is only available to {props.meta.resource_type}</span>.
```

Comment:
The `{props.meta.resource_type}` part isn't rendering correctly for some reason -- do you know why?

Here are the previews that I'm looking at:

- [models](https://docs-getdbt-com-git-mwong-config-descriptions-dbt-labs.vercel.app/reference/model-configs#available-configurations)
- [seeds](https://docs-getdbt-com-git-mwong-config-descriptions-dbt-labs.vercel.app/reference/seed-configs#available-configurations)
- [snapshots](https://docs-getdbt-com-git-mwong-config-descriptions-dbt-labs.vercel.app/reference/snapshot-configs#available-configurations)
- [sources](https://docs-getdbt-com-git-mwong-config-descriptions-dbt-labs.vercel.app/reference/source-configs#available-configurations)
- [tests](https://docs-getdbt-com-git-mwong-config-descriptions-dbt-labs.vercel.app/reference/test-configs#available-configurations)

<img width="500" alt="image" src="https://github.com/dbt-labs/docs.getdbt.com/assets/44704949/9742021b-a68a-4b8c-b357-6195258d2009">